.github/workflows/kind-verify-attestation.yaml

```diff
@@ -112,6 +112,10 @@ jobs:
         echo -n 'foobar e2e test' > ./predicate-file
         ./cosign attest --predicate ./predicate-file --fulcio-url ${{ env.FULCIO_URL }} --rekor-url ${{ env.REKOR_URL }} --allow-insecure-registry --yes ${{ env.demoimage }} --identity-token ${{ env.OIDC_TOKEN }}
 
+    - name: Sign a blob
+      run: |
+        ./cosign sign-blob README.md --fulcio-url ${{ env.FULCIO_URL }} --rekor-url ${{ env.REKOR_URL }} --output-certificate cert.pem --output-signature sig --yes --identity-token ${{ env.OIDC_TOKEN }}
```

Comment:
thanks! does this belong here instead? https://github.com/sigstore/cosign/blob/48e6404789cc34addfe15d0d6df01c23e529e034/.github/workflows/e2e-with-binary.yml

Reply:
Added there as well.